_includes/android/push-notifications.md

```diff
@@ -8,29 +8,7 @@ If you haven't installed the SDK yet, [head over to the Push QuickStart]({{ site
 
 If you want to start using push, start by completing the [Android Push Notifications QuickStart Guide]({{ site.baseUrl }}/parse-server/guide/#push-notifications-quick-start) to learn how to configure your app and send your first push notification. Come back to this guide afterwards to learn more about the push features offered by Parse.
 
-The Parse library provides push notifications using Google Cloud Messaging (GCM) if Google Play Services are available. Learn more about Google Play Services [here](https://developers.google.com/android/guides/overview).
-
-When sending pushes to Android devices with GCM, there are several pieces of information that Parse keeps track of automatically:
-
-*   **Registration ID**: The GCM registration ID uniquely identifies an app/device pairing for push purposes.
-*   **Sender ID**: The GCM sender ID is a public number that identifies the sender of a push notification.
-*   **API key**: The GCM API key is a server secret that allows a server to send pushes to a registration ID on behalf of a particular sender ID.
-
-The Parse Android SDK now requires you to specify the GCM sender ID with the following `<meta-data>` tag as a child of the `<application>` element in your app's `AndroidManifest.xml`:
-
-```java
-<meta-data android:name="com.parse.push.gcm_sender_id"
-           android:value="id:YOUR_SENDER_ID" />;
-```
-
-In the sample snippet above, `YOUR_SENDER_ID` should be replaced by a numeric GCM sender ID. Note that the Parse SDK expects you to prefix your sender ID with an `id:` prefix, as shown in the sample snippet.
-
-If you want to register your app with multiple additional sender IDs, then the `android:value` in the `<meta-data>` element above should hold a comma-delimited list of sender IDs, as in the following snippet:
-
-```java
-<meta-data android:name="com.parse.push.gcm_sender_id"
-           android:value="id:YOUR_SENDER_ID_1,YOUR_SENDER_ID_2,YOUR_SENDER_ID_3" />;
-```
+The Parse library provides push notifications using Firebase Cloud Messaging (FCM) if Google Play Services are available. Learn more about Google Play Services [here](https://firebase.google.com/docs/cloud-messaging/).
```

Comment:
I think this section should include the fact that calling ParseFCM.register(this) automatically passes the Installation object to the Parse server as a GCM token type.

Reply:
The user should never have to call this manually given our setup. Do you think we should still mention that it is happening?